adafruit_magtag/peripherals.py

```diff
@@ -143,3 +146,11 @@ def any_button_pressed(self):
         Return whether any button is pressed
         """
         return False in [self.buttons[i].value for i in range(0, 4)]
+
+    @property
+    def light(self):
+        """
+        Return the value of the light sensor. The neopixel_disable property
+        must be false to get a value.
+        """
```

Comment:
Consider adding the example to this  docstring.
```suggestion
        .. code-block:: python
            import time
            from adafruit_magtag.magtag import MagTag

            magtag = MagTag()

            while True:
                print(magtag.peripherals.light)
                time.sleep(0.01)
        """
```